cloudflare-worker/src/index.ts

```
@@ -0,0 +1,137 @@
+      if (request.method === 'POST' && url.pathname === '/cleanup') {
+        const { keepIds } = (await request.json()) as { keepIds: string[] };
+        if (!Array.isArray(keepIds)) {
+          return jsonResponse({ success: false, error: 'keepIds array required' }, 400);
+        }
+        const keepSet = new Set(keepIds);
+        const info = await env.VECTORIZE.describe();
+        if ((info.vectorCount ?? 0) === 0) {
+          return jsonResponse({ success: true, deleted: 0 });
+        }
+        // 使用 listVectors 分页枚举所有向量 ID，找出不在 keepSet 中的
+        let staleIds: string[] = [];
```

ESLint

[error] 'staleIds' is never reassigned. Use 'const' instead.

(prefer-const)